app/src/docker/docker-compose.v2.yml

```diff
@@ -0,0 +1,26 @@
+version: '3.9'
+
+networks:
+  internal:
+  external:
+
+services:
+  assess-claim-dc6602v2:
+    image: va/abd_vro-assessclaimdc6602v2:latest
+    build: ../../../service-python/assessclaimdc6602v2/src/.
+    environment:
+      RABBITMQ_PORT: 5672
+      RABBITMQ_RETRY_LIMIT: 3
+      RABBITMQ_TIMEOUT: 10800
+      RABBITMQ_PLACEHOLDERS_HOST: rabbitmq-service
+      RABBITMQ_PLACEHOLDERS_USERNAME: ${RABBITMQ_PLACEHOLDERS_USERNAME}
+      RABBITMQ_PLACEHOLDERS_USERPASSWORD: ${RABBITMQ_PLACEHOLDERS_USERPASSWORD}
+    security_opt:
+      - no-new-privileges:true
+    depends_on:
+      rabbitmq-service:
+        condition: service_healthy
+    networks:
+      - internal
+    restart: on-failure:5
```

Comment:
I removed this from others. Docker compose is only for development so we do not really need it.